Firmware/first_lay_cal.cpp

```diff
@@ -210,41 +224,37 @@ void lay1cal_meander(float layer_height, float extrusion_width)
 //! @param i iteration
 void lay1cal_square(uint8_t step, float layer_height, float extrusion_width)
```

Comment:
step parameter is now useless. Please remove it and fix the for loop to 4 cycles. Update comment above also and save some more flash by reducing parameter count :>

Reply:
done, saved 74B of flash